testutil/duration.go

```diff
@@ -9,10 +9,11 @@ import (
 // Constants for timing out operations, usable for creating contexts
 // that timeout or in require.Eventually.
 const (
-	WaitShort     = 10 * time.Second
-	WaitMedium    = 15 * time.Second
-	WaitLong      = 25 * time.Second
-	WaitSuperLong = 60 * time.Second
+	WaitSuperShort = time.Second
```

Comment:
You're right; I added this in because some log lines were spewing too much during a test (I added logging when no rows are found), but it'll be safer to just set an appropriate log level.

10 secs felt a bit too long as well, but I guess we have this in a bunch of places so it's fine.